packages/pulseaudio.rb

```diff
@@ -38,15 +25,12 @@ class Pulseaudio < Package
   depends_on 'tdb'
 
   def self.build
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}"
+    system "./configure #{CREW_OPTIONS}"
```

Comment:
Good thinking to switch to `#{CREW_OPTIONS}`.  This makes the package more portable.